.jenkins

```diff
@@ -2,6 +2,10 @@ pipeline {
 
   agent any
```

Comment:
This will need to be -eventually- dealt with. but not this pr.

Reply:
aaaagreed. 

Feels like we'll be able to have a similar pipeline for this + vapor-charts, so thats nice at least.